### ISSUE TITLE
Exercises for ml chapters

### ISSUE THREAD

**Issue by andreitsev** [mml-book/mml-book.github.io#488]
The book is great, but it would be even better if there were also exercises for machine learning chapters (such as linear regression, PCA, SVM and so on). Espesially practical problems would be appreciated. Such as, given some small dataset example, perform PCA on such a dataset... Something in that manner.

**Comment by mpd37** [comment]
We have jupyter notebook tutorial for the applied chapters. Is this what you were looking for?

**Comment by andreitsev** [comment]
Not quite, I was looking for problems, more like in classical subjects, such as calculus and others. For example: given a dataset (1, 1, 3), (-1, 2, 2), ... perform PCA on such a dataset. Key point is, problems should be such, that they should be done by hands, without need to use a computer



Отправлено из мобильной Почты Mail.ru


вторник, 3 марта 2020 г., 11:09 +0300 от notifications@github.com  <notifications@github.com>:

**Comment by mpd37** [comment]
I see. We decided not to do add exercises like this. For linear regression or PCA this would have been possible, but not for GMMs or SVMs.
